elasticsearch/config/load_settings.sh

```diff
@@ -23,6 +23,25 @@ done
 
 >&2 echo "Elastic is up - executing command"
 
+if [ $ENABLE_CONFIGURE_S3 ]; then
+
+  sleep 10
```

Comment:
Add a comment explaining why we need this sleep.

Reply:
¿Is 10 seconds enough for it?